src/klein/resource.py

```diff
@@ -280,3 +280,39 @@ def write_response(r):
         d.addCallback(write_response).addErrback(log.err, _why="Unhandled Error writing response")
 
         return server.NOT_DONE_YET
+
+
+
+class KleinHTTPRequest(server.Request):
+
+    def getArg(self, key):
```

Comment:
I really don't like the name `getArg`.  "args" are an incredibly ill-defined mess in Twisted, mixing together query parameters ("GET args") and URL-encoded or multipart form-values in the body ("POST args" or "form args").  If we're going to try to do something better, it would be nice to clean this up a bit.  Even if it is accessing the underlying `.args`, I feel like we should be disambiguating these namespaces.